src/delegation/DelegationRootNode.spec.ts

```diff
@@ -11,31 +9,25 @@ describe('Delegation', () => {
   let identityAlice: Identity
   let ctypeHash: string
   let ROOT_IDENTIFIER: string
-  const registry = new TypeRegistry()
 
   beforeAll(async () => {
     identityAlice = await Identity.buildFromURI('//Alice')
     ctypeHash = Crypto.hashStr('testCtype')
     require('../blockchainApiConnection/BlockchainApiConnection').__mocked_api.query.delegation.root.mockReturnValue(
-      new Option(
-        registry,
-        Tuple.with(
-          // Root-Delegation: root-id -> (ctype-hash, account, revoked)
-          ['H256', AccountId, Bool]
-        ),
-        [ctypeHash, identityAlice.getAddress(), false]
-      )
+      mockChainQueryReturn('delegation', 'root', [
+        ctypeHash,
+        identityAlice.getAddress(),
+        false,
+      ])
     )
     require('../blockchainApiConnection/BlockchainApiConnection').__mocked_api.query.delegation.delegations.mockReturnValue(
-      new Option(
-        registry,
-        Tuple.with(
-          // Root-Delegation: delegation-id -> (root-id, parent-id?, account, permissions, revoked)
-
-          ['H256', AccountId, Bool]
-        ),
-        [ctypeHash, identityAlice.getAddress(), false]
```

Comment:
Wondering why this worked out in the first place. The mock here was based on `root` and should not have worked for `delegations`.